docker-compose.aviva-sxa.yml

```diff
@@ -0,0 +1,18 @@
+# Overlay file for Aviva Solutions SXA specific Docker image names
```

Comment:
Public?

Reply:
Zag geen andere handige plek... suggesties?

Reply:
Andere repo en dan script om te kopieren en te bouwen of zo?